eip_operator/src/controller/eip.rs

```diff
@@ -87,6 +88,42 @@ impl k8s_controller::Context for Context {
             }
         };
         crate::eip::set_status_created(&eip_api, name, &allocation_id, &public_ip).await?;
+
+        if eip.status.as_ref().is_some_and(|s| s.resource_id.is_some()) {
+            // do nothing
+        } else {
+            let resource_api = eip.get_resource_api(&client);
+            let matched_resources = resource_api
+                .list(&eip.get_resource_list_params())
+                .await?
+                .items;
+            info!(
+                "Eip apply for {} Found matched {} resources",
+                name,
+                matched_resources.len()
+            );
+            for resource in matched_resources {
+                info!(
+                    "Updating eip refresh label for {}",
+                    resource.name_unchecked()
+                );
+                let data = resource.clone().data(serde_json::json!({
+                    "metadata": {
+                            "labels":{
+                               "eip.materialize.cloud/refresh": SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_secs().to_string()
```

Comment:
what is the purpose of this annotation? if this is just to force the pod/node resource to re-reconcile, can we use a random value here rather than a timestamp? it feels like that would be less likely to have weird collision behavior